src/filters/write_generic.c

```diff
@@ -937,6 +941,57 @@ static GF_Err ttml_embed_data(GF_XMLNode *node, u8 *aux_data, u32 aux_data_size,
 	return GF_OK;
 }
 
+
+//from filter txtin
```

Comment:
we should avoid duplicating these - maybe add generic ttml timestamp rewrite in src/media_tools/mdia_import.c ?

Reply:
Fine with me - just add a note also in load_text that the any changes in the ttml rewrite code should be reflected in write_generic